### ISSUE TITLE
Not working with cordova-sqlite adapter

### ISSUE THREAD

**Issue by matejuh** [nolanlawson/pouchdb-quick-search#74]
It seems that this plugin is not working with pouchdb-adapter-cordova-sqlite and cordova-sqlite adapter. When websql adapter is used during in browser development, search is working as expected. But when deployed to the device and cordova-sqlite adapter is used, no results are found although records in the database exist (allDocs returns results).

**Comment by nolanlawson** [comment]
Yeah this module needs to update to the latest `pouchdb-abstract-mapreduce` in order to work properly. Unfortunately it's a breaking change because it will start creating design documents.

**Comment by JPOak** [comment]
This is an old thread, so checking in to see if it is still true? Doing some research on using Pouch, Sqlite adapter for persistent storage, and some sort of search capability. Thxs.
